pagopa/io-app
# feat: [IGP-83] Fix "share" button label being trimmed on some Android devices

## Short description
This PR fixes the "share" button label being trimmed on some Android devices, when using Italian translations.

## List of changes proposed in this pull request
- `ts/features/bonus/bonusVacanze/components/buttons/ButtonConfigurations.ts`: `buttonFontSize` property exposed
- `ts/features/messages/components/MessageAttachmentPreview.tsx`: footer button sizes scaled down from standard to small when on Android

## How to test
Using the dev server with a configuration that generates messages with attachments, navigate to such message, select an attachment and:
- when on Android, check that the "Condividi" button label is not trimmed anymore
- when on Android, check that the other two buttons have the same label size as the "Condividi" button
- when on iOS, there is just one button with a standard font size label
